packages/pharos-cli/src/cmds/pharos-component.ts

```diff
@@ -71,6 +72,7 @@ const createPharosComponent = async (componentName: ComponentName): Promise<void
   if (!wasError) {
     console.log(colors.yellow(`\nAdding Pharos${titleCaseName} component export`));
     addComponentToIndexFile(currentDirectory, nameOptions);
+    addComponentsToInitFile(currentDirectory, nameOptions);
```

Comment:
Do we want to add another similar log, or generalize the existing one a bit?

Reply:
Good idea. Will add another new log that logs the message clearly about adding components to init files.

Reply:
Added new log with color `green`. Let me know if you feel any other color would be apt @daneah

Reply:
I think yellow like the existing log would be good. Green indicates success, but the log is saying something is about to happen, not that it already succeeded.

Reply:
updated log to yellow. So, resolving the conversation.